source/_integrations/zwave_me.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+featured: true
+ha_category:
+  - Binary Sensor
+  - Climate
+  - Cover
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
+  - Sensor
+  - Siren
+  - Switch
+ha_release: '2021.12'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lawfulchaos, @Z-Wave-Me'
+ha_domain: zwave_me
+ha_platforms:
+  - binary_sensor
+  - climate
+  - cover
+  - fan
+  - light
+  - lock
+  - number
+  - sensor
+  - select
+  - siren
+  - switch
+---
+
+This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). This is the recommended Z-Wave integration for Home Assistant for owners of Z-Wave.Me controllers.
+
+## Quick start (Home Assistant including Supervisor)
+
+To add Z-Wave.Me to your installation, go to {% my integrations title="Configuration > Devices & Services" %} in the UI. Click the "Add integration" button in the bottom right and from the list of integrations, select "Z-Wave.Me" and follow the instructions shown.
```

Comment:
We can use the config flow template here which will provide this text and more for free.
```suggestion
{% include integrations/config_flow.md %}
```